### PR TITLE
dx-1212 Consistent payer terminology part 1

### PR DESCRIPTION
**Pull request by arebra**
changed end user, end-user and consumer to payer except where it is supposed to be consumer. Flowcharts not included.

### CODE REVIEW

File: _includes/card-purchase.md [hunk 1]
```diff
@@ -121,17 +121,17 @@ Content-Type: application/json
...
+|                  | └➔&nbsp;`paymentToken`            | `string`      | If you enter a paymentToken here, the payment page will preload the stored payment data related to the `paymentToken`, and the payer can complete the purchase without having to enter all card data. This is called a "One Click" purchase.                                                                                                                                                                                                                                                                                                                                                 |
```
Comment (asbjornu) [inline _includes/card-purchase.md:124]: This sentence reads very strange to me and "the payment page will reload" is blatantly false, isn't it?

File: payment-menu/payment-order.md [hunk 2]
```diff
@@ -169,8 +169,8 @@ order.
 Below, you will see a complete overview of the payment menu process.
 Notice that there are two ways of performing the payment:
 
-*   Consumer perform payment **out** of `iframe`.
-*   Consumer perform payment **within** `iframe`.
+*   Payer perform payment **out** of `iframe`.
+*   Payer perform payment **within** `iframe`.
```
Comment (asbjornu) [inline payment-menu/payment-order.md:173]: "Perform" or "performs"?

File: payments/vipps/seamless-view.md [hunk 3]
```diff
@@ -296,14 +296,14 @@ sequenceDiagram
 2.  `rel: view-payment` is a value in one of the operations, sent as a response
    from Swedbank Pay to the Merchant.
 3.  `Open iframe` creates the Swedbank Pay hosted iframe.
```
Comment (asbjornu) [inline payments/vipps/seamless-view.md:298]:
```suggestion
3.  `Open iframe` creates the Swedbank Pay Seamless View.
```

File: payments/vipps/seamless-view.md [hunk 4]
```diff
@@ -296,14 +296,14 @@ sequenceDiagram
...
-     The consumer UI page displays the payment window as content inside of the
-   `iframe`. The consumer can insert mobile information for authorization.
+     The payer UI page displays the payment window as content inside of the
+   `iframe`. The payer can insert mobile information for authorization.
```
Comment (asbjornu) [inline payments/vipps/seamless-view.md:300]:
```suggestion
     The Seamless View displays the payment page as content inside of the
   `iframe`. The payer can enter their mobile phone number for authorization.
```

File: resources/data-protection.md [hunk 5]
```diff
@@ -8,11 +8,11 @@ estimated_read: 30
...
 create a profile.
 This will store certain data (listed below) to their profile.
-However if a consumer choose to not create a profile we store the consumer data
+If a consumer chooses to not create a profile, we store the consumer data
 as the provided data is only possible to input during this stage.
 To have access to this when completing a payment order, such as capturing it
-when shipping the order the address and name must be available on a profile,
-to achieve this we store the information temporarily to make sure it is available.
+when shipping the order the address and name must be available on a profile.
+&o achieve this we store the information temporarily to make sure it is available.
```
Comment (asbjornu) [inline resources/data-protection.md:15]:
```suggestion
To achieve this we store the information temporarily to make sure it is available.
```

**Review by asbjornu — CHANGES_REQUESTED** [review]
This is absolutely amazing! 👏 🎉

File: _includes/card-purchase.md (hunk 1 above)
Comment (arebra) [reply _includes/card-purchase.md:124]: It says preload, not reload :) If you enter a paymentToken which has been "activated" by a previous payment, the card will appear masked as a stored card, with the CVC as the only field that needs input.

But I agree that the wording is a bit awkward. I can re-phrase it for sure.

File: payment-menu/payment-order.md (hunk 2 above)
Comment (arebra) [inline payment-menu/payment-order.md:173]:
```suggestion
*   Payer performs payment **within** `iframe`.
```

File: payment-menu/payment-order.md [hunk 6]
```diff
@@ -169,8 +169,8 @@ order.
 Below, you will see a complete overview of the payment menu process.
 Notice that there are two ways of performing the payment:
 
-*   Consumer perform payment **out** of `iframe`.
-*   Consumer perform payment **within** `iframe`.
+*   Payer perform payment **out** of `iframe`.
```
Comment (arebra) [inline payment-menu/payment-order.md:172]:
```suggestion
*   Payer performs payment **out** of `iframe`.
```

File: payment-menu/payment-order.md (hunk 2 above)
Comment (arebra) [reply payment-menu/payment-order.md]: Performs. Already commited changes 👍

File: _includes/card-purchase.md (hunk 1 above)
Comment (arebra) [reply _includes/card-purchase.md:124]: If I understood your question correctly?

Comment (asbjornu) [reply _includes/card-purchase.md:124]: I see. "Prefilled" is a term used elsewhere, so perhaps that's better?

Comment (asbjornu) [reply _includes/card-purchase.md:124]: Will you rephrase it in this PR or in another?

Comment (arebra) [reply _includes/card-purchase.md:124]: I'll fix it here 👍

Comment (arebra) [inline _includes/card-purchase.md:124]:
```suggestion
|                  | └➔&nbsp;`paymentToken`            | `string`      | If a `paymentToken` is included in the request, the card details stored in the `paymentToken` will be prefilled on the payment page. The payer still has to enter the `cvc` to complete the purchase. This is called a "One Click" purchase.                                                                                                                                                                                                                                                                                                                                                 |
```

Comment (arebra) [reply _includes/card-purchase.md:124]: Better? :)

**Review by asbjornu — APPROVED** [review]

File: _includes/card-purchase.md (hunk 1 above)
Comment (asbjornu) [reply _includes/card-purchase.md]: Much! 👏

**Comment by asbjornu** [comment]
Just rebase this on a fresh `develop` and it's good to go! 👍

**Pull request merged**